<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldGenWindow</class>
 <widget class="QMainWindow" name="WorldGenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="WorldGen::WorldGenView" name="view"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIncreaseDepth"/>
    <addaction name="actionDecreaseDepth"/>
   </widget>
   <widget class="QMenu" name="menuView_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowGrids"/>
    <addaction name="actionOsm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuView_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionIncreaseDepth">
   <property name="text">
    <string>Increase Depth</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionDecreaseDepth">
   <property name="text">
    <string>Decrease Depth</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionShowGrids">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid(s)</string>
   </property>
  </action>
  <action name="actionOsm">
   <property name="text">
    <string>osm</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorldGen::WorldGenView</class>
   <extends>QGraphicsView</extends>
   <header>worldgenview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
